controllers/apply-next/simple/fields.js

```diff
@@ -622,23 +622,39 @@ module.exports = function fieldsFor({ locale, data = {} }) {
                     fromDateExample: dt
                         .subtract(1, 'days')
                         .format('D MMMM YYYY'),
-                    minYear: dt.format('YYYY')
+                    minYear: dt.format('YYYY'),
+                    // @TODO should there be a human readable version of this
+                    // (eg. "one year") for labels/explanations?
+                    maxDurationFromStart: {
+                        amount: 1,
+                        units: 'years'
+                    }
                 };
             },
             get explanation() {
                 return localise({
-                    en: `<p>This date needs to be at least 12 weeks from when you plan to submit your application. If your project is a one-off event, please tell us the date of the event.</p>
-                <p><strong>For example: ${
-                    this.settings.minDateExample
-                }</strong></p>`,
+                    en: `<p>The start date needs to be at least 12 weeks from when you plan to submit your application. If your project is a one-off event, please tell us the date of the event.</p>
+                    <p><strong>For example: ${
+                        this.settings.minDateExample
+                    }</strong>. The project end date must be within ${
```

Comment:
I'd either write this in words (like the "12 weeks above") or format this using moment earlier on (like in your TODO comment)

Reply:
Done, ended up making the "12 weeks" bit dynamic too so we don't inadvertently break that down the line.